bson/src/test/unit/org/bson/types/ObjectIdTest.java

```diff
@@ -88,12 +89,12 @@ public void testDateCons() {
 
     @Test
     public void testMachineIdentifier() {
-        assertTrue(ObjectId.getGeneratedMachineIdentifier() > 0);
+        assertTrue(ObjectId.getGeneratedMachineIdentifier() == 0);
```

Comment:
I think we'll be able to change this back

Reply:
I have changed this back.